packages/@office-iss/react-native-win32/src/Libraries/Components/View/ViewWin32.Props.ts

```diff
@@ -178,6 +178,12 @@ export type BasePropsWin32 = {
     * ItemType is used to obtain information about items in a list, tree view, or data grid. For example, an item in a file directory view might be a "Document File" or a "Folder".
     */
    accessibilityItemType?: string;
+
+   /**
+    * Defines the level of an element in a hierarchiacal structure or the heading level of a text element.
+    * Note: accessibilityRole="header" must be used if using this property to define a heading level.
+    */
+    accessibilityLevel?: number;
```

Comment:
Does it make sense for accessibilityLevel to be added here if it's constrained to Text? Is there a scenario where a View would want accessibilityLevel?

Reply:
Yes, accessibilityLevel can be used to describe the location of an element inside a hierarchy. For example a bulleted/numbered list, tree, menu or other structure with parent/child relationships.

Reply:
Should a test case be added for that case?

Reply:
Yeah, that's a good idea. Just added one and verified that a text component w/o the "header" role set does not get read as a header and no heading level is set.

There are also already test cases showing accessibilityLevel working in the same file (see MultiSelection and FlatList examples).

Reply:
Ah got it, thanks for clarifying. Looks good to me!